src/components/EventCreationPage/EventCreationPage.tsx

```diff
@@ -59,7 +61,11 @@ const EventCreationPage = ({
       history.replace('/events');
     },
   });
-  const [editEvent] = useMutation(EDIT_EVENT);
+  const [editEvent] = useMutation(EDIT_EVENT, {
+    onCompleted() {
+      history.replace('/events');
```

Comment:
I moved the `history.replace` function to up here, because what was happening before was that we were redirecting back to `/events` before the `addEvent` mutation finished running